Jenkins/PRChecks/Jenkinsfile

```
@@ -0,0 +1,136 @@
+pipeline {
+    agent {
+        node {
+            label 'docker && CI-role && !master && !gherkin'
+        }
+    }
+    options {
+        disableConcurrentBuilds(abortPrevious: true)
+        timeout(time: 2, unit: 'HOURS') 
+    }
+    environment {
+        TOKEN = credentials('studocu-ci-status-access-token')
+        GITHUB_TOKEN = credentials('jenkins-sonar-gh-token')
+        PR_FOUND = sh(returnStatus: true, script: '!(gh pr view $BRANCH_NAME --json labels -q ".labels.[].name" | grep "Ready for Tests")')
+        PR_ID = sh(returnStdout: true, script: 'gh pr view $BRANCH_NAME | grep \'number\' | awk \'{print $2}\'').trim();
+        DOCKER_NODE_IMAGE = '388073764231.dkr.ecr.eu-west-1.amazonaws.com/node:25076521f02448d3bf0b5a8cbdf18602326fb39c'
+    }
+    stages {
+        stage('Run PR Checks') {
+            when {
+                beforeAgent true
+                anyOf {
+                    branch 'master'
```

```suggestion
                    branch 'main'
```

same for any other mention of `master` since we're merging to `main`

---

[bf0a2ee](https://github.com/StuDocu/design-figma-tokens/pull/19/commits/bf0a2eefd4ad60a8a01ba823aa49aca44e01e590)